README.md

```diff
@@ -22,31 +22,61 @@ If you want to read further click [here](https://www.workamajig.com/blog/guide-t
 
 ## Coding Standards
 
-### General Programming Practices
-
 It is important to realise that your programming choices will reflect the quality
-of your code and impact the productivity levels of your colleagues. 
+of your code and impact the productivity levels of your colleagues. Multidots - an agency specialising in web development for content publishers [coding standards guideline](https://www.multidots.com/importance-of-code-quality-and-coding-standard-in-software-development/) highlights the importance of code quality and coding standards for improving efficiency, minimising risk of project failure, reducing complexity and simplifying debugging processes.
 
-The following guidelines 
-are recommended for best coding practices to be used when working on projects:
+### General Programming Practices
 
 #### Naming Conventions
 
-Use meaningful variable names throughout your code. Maintain a consistent naming style for classes, functions and files. Names should be explanatory of the object or the functionality of the methods that carry them. 
+* Use meaningful variable names throughout your code
+* Maintain a consistent naming style for classes, functions and files
+* Names should be explanatory of the object or the functionality of the methods that carry them. 
 
 #### Formatting and Layout
 
-Code formatting and layout
-Code segmentation & indentation for readabilit
+To work together as a team, it is important that each developer effectively formats code,  tools and ensures code is well-segmented and easy to read. 
+
+* Use auto-lint provided by the development environment your team is using
+* Use block separation to make files more readable
+
+For all programming languages refer to general [guidelines](https://firefox-source-docs.mozilla.org/code-quality/coding-style/index.html) for coding and formatting styles recommended by the Firefox development team.
+
+### Developement Style
+
+**Reusability**
```

Comment:
For code reusability, you could use some code example as images.

Visualization helps readers keep focused. I have some examples in this link here
https://www.javaworld.com/article/2077470/java-tip-107--maximize-your-code-reusability.html